games-strategy/hedgewars/hedgewars-0.9.25.ebuild

```diff
@@ -0,0 +1,115 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+CMAKE_MAKEFILE_GENERATOR=emake
+inherit cmake-utils desktop xdg-utils
+
+MY_P=${PN}-src-${PV}
+DEB_PATCH_VER=2
+
+DESCRIPTION="A turn-based strategy, artillery, action and comedy game"
+HOMEPAGE="https://www.hedgewars.org/"
+SRC_URI="https://www.hedgewars.org/download/releases/${MY_P}.tar.bz2
+	mirror://debian/pool/main/h/${PN}/${PN}_0.9.23-dfsg-${DEB_PATCH_VER}.debian.tar.xz"
+LICENSE="GPL-2 Apache-2.0 FDL-1.3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="libav server pas2c"
+
+REQUIRED_USE="
+	x86? ( pas2c )
```

Comment:
If you do that, pas2c should be enabled by default not to cause QA warnings for `x86`.

Wild idea: If we unconditionally apply pas2c for everyone, could we drop the `QA_` quirks below?

Reply:
Here is an irc log from #hedgewars on freenode on 2018-12-03 than can help: https://ptpb.pw/Xa2F.log

Reply:
irc logs from 2018-11-21 are also interesting:
https://ptpb.pw/hBcd.log

Reply:
There is an official explaination on upstream blog post here: http://hedgewars.org/node/7028

Reply:
But the `pas2c` USE flag can be used in other architectures as upstream is planning to deprecate fpc usage in their build recommandations. Then people wanting to use it can help them improving the process.

Reply:
That's not what I read from the last two paragraphs of the blog post.